src/main/java/com/olgaboiar/mint/Logger.java

```diff
@@ -0,0 +1,23 @@
+package com.olgaboiar.mint;
+
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.IOException;
+
+public class Logger {
```

Comment:
Test?

Reply:
added in https://github.com/olgaboiar/MintHTTPServer/pull/6/commits/a9a24390b3810ef211c4424eeb4ec8111eeb3f3b